gcc/testsuite/rust/compile/generics6.rs

```diff
@@ -24,8 +24,6 @@ impl Foo<f32> {
 
 fn main() {
     let a: i32 = Foo::test(); // { dg-error "multiple applicable items in scope for: test" }
-    // { dg-error {failed to type resolve expression} "" { target *-*-* } .-1 }
-    // { dg-error {Failed to resolve expression of function call} "" { target *-*-* } .-2 }
-    // { duplicate _dg-error {failed to type resolve expression} "" { target *-*-* } .-3 }
+    // { dg-error {Failed to resolve expression of function call} "" { target *-*-* } .-1 }
```

Comment:
In most cases, I can resolve the issue by simply deleting the line `{ dg-error {failed to type resolve expression} ...`. However, in three instances, there were duplicate tests that required additional attention. Although I was able to pass the `make check-rust` test in my local environment, I am uncertain whether or not my solution is entirely correct.

Reply:
The dg annotations require the -1 or -2 etc at the end to match up to line numbers in the location info. This looks good to me